packages/foreman/rubygem-sprockets/rubygem-sprockets.spec

```diff
@@ -0,0 +1,105 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name sprockets
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.7.2
+Release: 4%{?dist}
+Summary: Rack-based asset packaging system
+Group: Development/Languages
+License: MIT
+URL: https://github.com/rails/sprockets
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 3.7.2
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9.3
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}rubygem(rack) > 1
+Requires: %{?scl_prefix_ruby}rubygem(rack) < 3
```

Comment:
This should be `scl_prefix` as we are building this package and not come from the Ruby SCL.

Reply:
There are other packages that require Rack and are having the same issue.